src/components/ApiComponent/api.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios'
+
+const API = axios.create({
```

Comment:
export a function which will directly make a request to our endpoint. Call it like `getTodos`